docs/audio.rst

```diff
@@ -240,13 +251,19 @@ Technical Details
     You don't need to understand this section to use the ``audio`` module.
     It is just here in case you wanted to know how it works.
 
-The ``audio`` module can consumes an iterable (sequence, like list or tuple, or
-generator) of ``AudioFrame`` instances, each 32 samples at 7812.5 Hz, and uses
-linear interpolation to output a PWM signal at 32.5 kHz, which gives tolerable
-sound quality.
+The ``audio.play()`` function can consume an instance or iterable
+(sequence, like list or tuple, or generator) of ``AudioFrame`` instances.
+Its default playback rate is 7812 Hz, and uses linear interpolation to output
+a PWM signal at 32.5 kHz.
```

Comment:
I think we should get rid of this linear interpolation, and just output the 7812 Hz signal directly.  I did some tests, and the audio quality is better *without* the interpolation.

Reply:
Sounds good, in that case let's do that and remove this from the docs 👍